### PR TITLE
Added a gobuster helper script

### PR DESCRIPTION
**Pull request by sm4rtshr1mp**

### CODE REVIEW

File: bin/ar-prune-blacklisted-domains [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #! /bin/bash
 
+if [ ! -f blacklist-domains.txt ]; then
```
Comment (defektive) [inline bin/ar-prune-blacklisted-domains:3]: should we check for zero length ` blacklist-domains.txt` as well? I am pretty sure that breaks it as well.

File: bin/arbuster [hunk 2]
```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
```
Comment (defektive) [inline bin/arbuster:1]: this looks awesome

**Review by defektive — COMMENTED** [review]
one question

File: bin/ar-prune-blacklisted-domains (hunk 1 above)
Comment (sm4rtshr1mp) [reply bin/ar-prune-blacklisted-domains:3]: Yeah I am pretty sure a zero length file would break it as well

**Review by defektive — APPROVED** [review]

**Pull request merged**